packages/suite/src/utils/wallet/__fixtures__/coinControlUtils.ts

```diff
@@ -0,0 +1,60 @@
+import * as coinControlUtils from '../coinControlUtils';
+
+const baseUtxo = {
```

Comment:
nit: I suggest to do declarative typing in cases like this. It has two benefits:

1) The autocomplete in IDEs works
2) When there is a type error, it is reported by Typescrpt in a place where the error originates and not where this  `baseUtxo` is used.